src/object.d

```diff
@@ -1740,6 +1740,16 @@ class Throwable : Object
             }
         }
     }
+
+    /**
+     * Get the message describing the error.
+     * Base behavior will be to return the `Throwable.msg` field.
+     * Override to return some other error message.
+     *
+     * Returns:
+     *  Error message
+     */
+    @__future const(char)[] message() const;
```

Comment:
OK, I guess this is one of the places where the implementation doesn't match the DIP... :unamused: